src/pages/privacy-policy.tsx

```diff
@@ -383,7 +383,7 @@ const Page = (): JSX.Element => (
           <ListItem>Telegram</ListItem>
           <ListItem>GitHub</ListItem>
           <ListItem>Google</ListItem>
-          <ListItem>Twitter</ListItem>
+          <ListItem>X</ListItem>
```

Comment:
Let's don't modify the privacy and terms and conditions pages

Reply:
true, it was a mistake, I didn't want to modify this file.